sirepo/package_data/static/css/raydata.css

```diff
@@ -0,0 +1,6 @@
+.raydata-overflow-text {
```

Comment:
@moellep If you have better suggestions for css please let me know. The most interesting case is when you expand the plan args field in the plan report and then adjust the browser window in the horizontal direction. In some cases a scroll bar is added so the content doesn't overflow the report panel. We don't use scrollbars in many places so if you'd like something different lmk.